pkg/cluster/nodes/create.go

```diff
@@ -141,8 +141,6 @@ func createNode(name, image, clusterLabel, role string, mounts []cri.Mount, extr
 		"--label", clusterLabel,
 		// label the node with the role ID
 		"--label", fmt.Sprintf("%s=%s", constants.NodeRoleKey, role),
-		// explicitly set the entrypoint
-		"--entrypoint=/usr/local/bin/entrypoint",
```

Comment:
we can remove this and the args below now that we have #468

this allows us to start a loadbalancer image identically to the other nodes